preferences.py

```diff
@@ -166,6 +166,11 @@ def find_installed_rendermans(self, context):
         subtype='FILE_PATH',
         default=os.path.join('$OUT', 'images', '{scene}.{layer}.{pass}.####.{file_type}'))
 
+    draw_ipr_text = BoolProperty(
+        name="Draw IPR Text",
+        description="Draw PRMan Interactive Mode Running when IPR is running",
```

Comment:
All I could quibble with is the description maybe "Draw notice on View3D when IPR is active?